wod_predictor/feature_engineering_parts/helpers.py

```diff
@@ -64,62 +64,6 @@ def fill_missing_knn(df, neighbors, data_columns=[]):
     return df_KNN
 
 
-def remove_outliers(df, method="iqr", score_headers: list = None, **kwargs):
```

Comment:
This was moved to a new class, with `fit` and `transform` methods separated out.